### PR TITLE
Modify diff_col feature to get the diff from sibling-start to node-start

### PR DESCRIPTION
**Pull request by warenlg**
Signed-off-by: Waren Long <waren@sourced.tech>

Fixed https://github.com/src-d/style-analyzer/issues/271

### CODE REVIEW

File: lookout/style/format/features.py [hunk 1]
```diff
@@ -125,8 +125,7 @@ class _FeatureDiffCol(OrdinalFeature):
 
     def __call__(self, sibling: Union[VirtualNode, bblfsh.Node], node: VirtualNode
                  ) -> Iterable[int]:
-        yield min(abs(sibling.start.col - node.end.col),
-                  abs(sibling.end.col - node.start.col))
+        yield abs(sibling.start.col - node.start.col)
```
Comment (m09) [inline lookout/style/format/features.py]: this leaks the node length when sibling is after the node. I think considering that we should just rename diff_col and not change the computation. Maybe col_gap?

**Comment by vmarkovtsev** [comment]
@m09 @warenlg I am confused, shall we merge this?

**Comment by warenlg** [comment]
Yes we should, I just raised an issue to remember @m09 to add a check to his PR related to these modified features as he asked me too.

**Comment by m09** [comment]
LGTM

**Pull request merged**